docs/REST.md

```diff
@@ -95,6 +95,31 @@ Retrieve a model.
 }
 ```
 
+## GET `/api/model/all-models`
```

Comment:
`all-models` is redundant, we already know its about models, so `all` would be better.